src/testing/testing.go

```diff
@@ -255,7 +258,28 @@ func (m *M) Run() int {
 	}
 
 	failures := 0
+
+	match := func(s string) bool { return true }
+
+	if flagRunRegexp != "" {
+		re, err := regexp.Compile(flagRunRegexp)
+		if err != nil {
+			fmt.Printf("testing: invalid regexp for -test.run: %v", err)
```

Comment:
Shouldn't this print to stderr?
Also, a possibly cheaper alternative is this:

```suggestion
			fmt.Println("testing: invalid regexp for -test.run:", err.Error())
```

Also, looking at that it appears that the line is incorrect anyway: it misses a newline at the end.

Reply:
Yes, regexps work (in general).  There's a Bus Error during `TestOnePassCutoff` I haven't investigated (I think it might be in `Compile` or `Simplify`, and the other test I need to disable is `TestRE2Exhaustive` because it just takes *so* long (and I'm assuming there isn't a bug that's causing an infinite loop -- again, haven't verified this).

The rest of the tests pass.  Although, annoying this PR introduces an import cycle when trying to test the regexp package!  Oh noes!  I wonder if we need a different fix..)

Reply:
>  Although, annoying this PR introduces an import cycle when trying to test the regexp package! Oh noes! I wonder if we need a different fix..

Upstream this is solved via the internal/testdeps package. I think we should be using that instead of importing the testing package directly.
(I haven't investigated what the state of internal/testdeps is in TinyGo).

Reply:
As for printing to stderr, I checked this and the standard Go toolchain prints this message to stdout.